src/lib/app_registry.tsx

```diff
@@ -91,7 +91,16 @@ class WorksForYou extends RootContainer {
   }
 }
 
+class MyProfile extends RootContainer {
```

Comment:
So, here we're talking about "my account" as it were. Is there a specific reason we're calling it profile? I know it's quite obvious here, but in Eigen for example we have so many references to profiles (in the context of metainformation, if I understood correctly). I'm wondering whether it might save us confusion in the future if we call it account or ... dno, something? 😅